jekyll/_cci2/about-circleci.md

```diff
@@ -1,56 +1,23 @@
 ---
 layout: classic-docs
-title: "About CircleCI"
-short-title: "About CircleCI"
+title: "Overview"
+short-title: "Overview"
 description: "Starting point for CircleCI 2.0 docs"
 categories: [getting-started]
 order: 1
 ---
 
-CircleCI is a modern **continuous integration** and **continuous delivery** platform.
+CircleCI is a modern continuous integration and continuous delivery (CI/CD) platform. The hosted solution is available at <https://circleci.com/>. The [CircleCI Enterprise](https://circleci.com/enterprise/) solution is installable inside your private cloud or data center. Both are free to try for two weeks. CircleCI automates build, test, and deployment of software artifacts to any service with an API or to any machine with `sshd`. 
 
-Our hosted solution is available at <https://circleci.com/>, or you can run [CircleCI Enterprise](https://circleci.com/enterprise/) behind the firewall, inside your private cloud or data center.
+![CircleCI Example Flow with GitHub]({{ site.baseurl }}/assets/img/docs/how_it_works.png)
 
-## Continuous Integration
+For example, after a software repository on GitHub or Bitbucket is authorized and added as a project to the circleci.com SaaS application, every new commit triggers a build and notification of success or failure through webhooks. CircleCI also supports Slack, HipChat, Campfire, Flowdock, and IRC notifications. Code coverage results are available from the details page for any project for which a reporting library is added.
 
-Continuous integration is a development practice where you regularly test all changes made to your codebase.
+CircleCI is then able to automatically deploy code to various environments based on successful tests. The following  infrastructure deployment mechanisms are integrated with CircleCI:
```

Comment:
Correct - this needs to be changed to: "CircleCI can be configured to deploy code to various environments, including the following:"